stories/config.js

```diff
@@ -19,6 +19,7 @@ installChartjs();
 
 addDecorator(initScreenshot());
 setScreenshotOptions({
+  waitFor: "waitFor",
```

Comment:
Maybe a quick comment to explain what we use this for?

Reply:
Also, just to confirm, is this just a no-op for screenshot tests that don't define `waitFor`?

Reply:
It is a no-op when `waitFor` is not defined